sympy/physics/mechanics/functions.py

```diff
@@ -381,6 +382,55 @@ def gravity(acceleration, *bodies):
 
     return gravity_force
 
+
+def center_of_mass(point, *bodies):
```

Comment:
Are there any tests for this?

Reply:
I haven't added a test for this. I shall do so in the next PR.

Reply:
This was also not addressed before merging.